src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -193,13 +193,21 @@ internal sealed partial class ConsoleHost
                 }
 
 #if !UNIX
-                // Creating a JumpList entry takes around 55ms when the PowerShell process is interactive and
-                // owns the current window (otherwise it does a fast exit anyway). Since there is no 'GET' like API,
-                // we always have to execute this call because we do not know if it has been created yet.
-                // The JumpList does persist as long as the filepath of the executable does not change but there
-                // could be disruptions to it like e.g. the bi-annual Windows update, we decided to
-                // not over-optimize this and always create the JumpList as a non-blocking background task instead.
-                Task.Run(() => TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator));
+                try
+                {
+                    // Creating a JumpList entry takes around 55ms when the PowerShell process is interactive and
+                    // owns the current window (otherwise it does a fast exit anyway). Since there is no 'GET' like API,
+                    // we always have to execute this call because we do not know if it has been created yet.
+                    // The JumpList does persist as long as the filepath of the executable does not change but there
+                    // could be disruptions to it like e.g. the bi-annual Windows update, we decided to
+                    // not over-optimize this and always create the JumpList as a non-blocking background task instead.
+                    Task.Run(() => TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator));
+                }
+                catch
```

Comment:
It does not work as you expect https://stackoverflow.com/questions/32067034/how-to-handle-task-run-exception

Reply:
Thanks. I did not think of that. Should we have the try catch inside the lambda expression then as we are not async here and also do not want to block.